src/Timeline/Timeline.js

```diff
@@ -0,0 +1,35 @@
+// @flow
+import React, { Component, type Node } from 'react';
+import Box from 'reakit/Box';
```

Comment:
You can now import `Box` from the `primitives` folder. I'm currently refactoring components to reference this folder in the TypeScript refactor.

```suggestion
import { Box } from '../primitives';
```